prediction_market_agent/agents/microchain_agent/omen_functions.py

```diff
@@ -33,5 +40,36 @@ def __call__(self) -> str:
         )
 
 
+class CreatePredictionMarket(Function):
+    # Hard-coded low value before it's tested out more and real use-case is required.
+    INITIAL_FUNDS = xdai_type(0.01)
+
+    @property
+    def description(self) -> str:
+        return f"Use this function to create a new prediction market on Omen. Question of the prediciton market can only be binary, in the Yes/No format. Using this function will cost you {CreatePredictionMarket.INITIAL_FUNDS} xDai."
+
+    @property
+    def example_args(self) -> list[str]:
+        return ["Binary yes/no question", "Closing time"]
```

Comment:
I would also add a proper example, e.g. "Will Trump win the election in 2024" and "2024-01-01" etc

Reply:
Sure, added